demo/default.css

```
@@ -108,10 +129,27 @@ code,
+	line-height: normal;
 	margin-bottom: 5px;
 	min-height: 15rem;
+	padding: 2px;
 	resize: vertical;
 	width: 100%;
+}
+
+#output-view {
+	overflow: auto;
+	white-space: pre-wrap;
+	word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)